src/DynamoCoreWpf/Manipulation/DynamoManipulatorContext.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Dynamo.Controls;
+
+namespace Dynamo.Wpf.Manipulation
+{
+    public class DynamoManipulatorContext
+    {
+        internal DynamoManipulatorContext() { }
+
+        public DynamoView View { get; set; }
```

Comment:
We should not pass out the `DynamoView` to client code like this. We should make the property immutable by only expose `{ get; }`.  

My suggestion is to leave this property out until we have more specific API requirements.